data/src/lib.rs

```diff
@@ -0,0 +1,82 @@
+//! The single source of truth for all data paths.
+//!
+//! Run `cargo make gen-assets` before using this!
+
+pub mod bench;
+pub mod example;
+pub mod ltr;
+pub mod qambert;
+pub mod smbert;
+pub mod test;
+
+use std::{
+    collections::HashMap,
+    env::{current_dir, var_os},
+    fs::File,
+    io::{BufReader, Error, ErrorKind, Result},
+    path::{Path, PathBuf},
+};
+
+use serde::Deserialize;
+use serde_json::{from_reader, from_value, Value};
```

Comment:
I personally prefer prefixing constructors with the type/mod/crate instead of importing them as this removes ambiguities.

E.g. I would use `serde_json::from_reader` and `serde_json::from_value` directly in the one place they appear in the code. If I feel `serde_json` is to long/repeats to often I maybe would reexport it as `json` and then use `json::from_reader`, `json::from_value`.

Reply:
i agree that this makes sense in a context with different constructors. in this case, those are the only constructors and there shouldn't be any danger of confusion. also hovering over the function shows the tooltip/library if needed for clarification.